qiskit/pulse/builder.py

```diff
@@ -2627,14 +2672,11 @@ def cx(control: int, target: int):  # pylint: disable=invalid-name
     call_gate(gates.CXGate(), (control, target))
 
 
+@deprecate_func(since="0.46.0")
```

Comment:
u1 should have a similar warning to the other gates.

Reply:
Applied suggestion :)